src/platform-implementation-js/platform-implementation.js

```diff
@@ -240,28 +239,17 @@ export function makePlatformImplementation(
       );
   }
 
-  const DRIVERS_BY_ORIGIN: { [name: string]: * } = {
-    'https://mail.google.com': GmailDriver,
-    'https://inbox.google.com': InboxDriver,
-  };
-
   const LOADER_VERSION: string = opts.VERSION;
   const IMPL_VERSION: string = BUILD_VERSION;
   const logger = new Logger(appId, opts, LOADER_VERSION, IMPL_VERSION);
 
   const origin: string =
     (process.env.NODE_ENV === 'test' && global.__test_origin) ||
     document.location.origin;
-  const DriverClass = DRIVERS_BY_ORIGIN[origin];
-  if (!DriverClass) {
-    console.log('InboxSDK: Unsupported origin', origin);
-    // never resolve
-    return new Promise((resolve, reject) => {});
-  }
```

Comment:
We need to keep the logic that if we're on an unrecognized domain we don't initialize the driver. It's common enough for people to import the InboxSDK in a content script that gets run on other domains. We could axe the console.log though: that was present to discourage people from remote-loading us into non-Gmail domains and giving us a large permission we didn't need, but we don't remote-load in the now recommended npm build so it doesn't matter much now.

Reply:
[b81ce52](https://github.com/InboxSDK/InboxSDK/pull/875/commits/b81ce52b738292abcda8e30f62783af49e68e199)

Reply:
> We need to keep the logic that if we're on an unrecognized domain we don't initialize the driver.

Keeping this logic has the added benefit of patching up #873.